axios.js

```diff
@@ -5,8 +5,8 @@ const apiClient = axios.create({
     timout: 5000,
 })
 
-export function fetchArticles () {
-    return apiClient.get(`/articles`)
+export function fetchArticles (query) {
+    return query? apiClient.get(`/articles?${query}`) : apiClient.get(`/articles`)
```

Comment:
axios get method takes a query object as the second argument. Take a look at that so you don't need to use ternary here